src/workspace-data/data-table/entity-service/EntityDeleter.js

```diff
@@ -52,7 +52,7 @@ export const EntityDeleter = ({ onDismiss, onSuccess, namespace, name, selectedE
       runningSubmissionsCount > 0 &&
         b({ style: { display: 'block', margin: '1rem 0' } }, [
           `WARNING: ${runningSubmissionsCount} workflows are currently running in this workspace. ` +
-            'Deleting the following data could cause failures if a workflow is using this data.',
+            'Deleting the following entities could cause workflows to fail if they are using the data.',
```

Comment:
As long as we're wordsmithing... I think it's not clear what "the data" refers to here. Is it the entities being deleted? The entire data table?

```suggestion
            'Deleting the following entities could cause workflows using them to fail.',
```